webdriver-spec.html

```diff
@@ -2835,6 +2859,25 @@
  <dfn data-lt="scroll into view|scrolls into view">scrolled into view</dfn>
  by calling <a>scrollIntoView()</a>.
 
+<p>An <a>element</a> is considered <dfn>editable</dfn> if it is
+ an <a><code>input</code> element</a>
+ whose <a><code>type</code> attribute</a> is not in the
+ <a>Button</a>,
+ <a>Hidden</a>,
+ <a>Image Button</a>,
+ <a>Reset Button</a>,
+ or <a>Submit Button</a> states,
+ is a <a><code>textarea</code> element</a>,
```

Comment:
Yes, it’s not very clear at the moment.  Tried to address this in fac6f9f.  Does it read better for you now?